rdt-open-reader/src/main/java/io/ona/rdt/job/RDTJobCreator.java

```diff
@@ -34,6 +37,8 @@ public Job create(@NonNull String tag) {
                 return new RDTSyncSettingsServiceJob();
             case LocationStructureServiceJob.TAG:
                 return new LocationStructureServiceJob();
+            case SyncAllLocationsServiceJob.TAG:
+                return new SyncAllLocationsServiceJob(RDTSyncAllLocationsIntentService.class);
```

Comment:
`SyncAllLocationsServiceJob` should be overridden, it does not run `RDTSyncAllLocationsIntentService` and instead runs `SyncAllLocationsIntentService` which is not what we want.